client-java/sql/src/main/java/org/evomaster/client/java/sql/QueryResultSet.java

```diff
@@ -0,0 +1,58 @@
+package org.evomaster.client.java.sql;
+
+import java.util.TreeMap;
+
+public class QueryResultSet {
+
+    private final TreeMap<String, QueryResult> queryResults;
```

Comment:
any specific reason to use a TreeMap? even if the concrete class is going to be a TreeMap, shouldn't the reference be a Map? also, need to add comment specifying what the String keys are

Reply:
I will improve the understability of the code.